### ISSUE TITLE
Fix typo in lxqt-admin/CMakeLists.txt

### ISSUE THREAD

**Issue by OlivierDuchateau** [lxde/lxqt-l10n#98]
There's a typo in lxqt-admin/CMakeLists.txt, when we want to disable one of the following options:
- WITH_LXQT_ADMIN_USER
- WITH_LXQT_ADMIN_TIME

Components are not reflected their options. Patch fixes this issue.

``` diff
From c9093f4724a771998a55cfc3bbc26c45e9e1e297 Mon Sep 17 00:00:00 2001
From: Olivier Duchateau <duchateau.olivier@gmail.com>
Date: Sun, 9 Oct 2016 11:28:45 +0200
Subject: [PATCH] Fix components and options

---
 lxqt-admin/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lxqt-admin/CMakeLists.txt b/lxqt-admin/CMakeLists.txt
index dfde3ea..1921945 100644
--- a/lxqt-admin/CMakeLists.txt
+++ b/lxqt-admin/CMakeLists.txt
@@ -6,5 +6,5 @@ option(WITH_LXQT_ADMIN_TIME "Build lxqt-admin-time translations" ON)
 option(WITH_LXQT_ADMIN_USER "Build lxqt-admin-user translations" ON)

 #components
-add_component(WITH_LXQT_ADMIN_USER lxqt-admin-time)
-add_component(WITH_LXQT_ADMIN_TIME lxqt-admin-user)
+add_component(WITH_LXQT_ADMIN_USER lxqt-admin-user)
+add_component(WITH_LXQT_ADMIN_TIME lxqt-admin-time)
-- 
2.9.2
```

**Comment by agaida** [comment]
GTM

**Comment by palinek** [comment]
fixed by #101
